style.css

```diff
@@ -52,4 +89,137 @@ header{
 }
 .front-face,.back-face{
     pointer-events: none;
+}
+
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
move this part to the top , after the *{ }

Reply:
> move this part to the top , after the *{ }

OK sure !